steward/src/somm_send.rs

```diff
@@ -60,7 +60,7 @@ async fn __send_messages(
     fee: Coin,
     messages: Vec<Msg>,
 ) -> Result<TxResponse, CosmosGrpcError> {
-    let cosmos_address = cosmos_key.to_address(&contact.get_prefix()).unwrap();
+    let cosmos_address = cosmos_key.to_address(&contact.get_prefix())?;
```

Comment:
We should still consider moving these repeated computations that are dependent on config values to config instantiation time.

Reply:
Created #84 to cover this